spec/infomon_spec.rb

```diff
@@ -115,12 +115,12 @@ def self.game
         Infomon::Parser.parse(line).eql?(:ok) or fail("did not parse:\n%s" % line)
       }
 
-      expect(Infomon.get("stat.fame")).to eq(4_804_958)
-      expect(Infomon.get("stat.fxp_current")).to eq(1_350)
-      expect(Infomon.get("stat.fxp_max")).to eq(1_010)
-      expect(Infomon.get("stat.ascension_experience")).to eq(4_170_132)
-      expect(Infomon.get("stat.total_experience")).to eq(41_307_131)
-      expect(Infomon.get("stat.long_term_experience")).to eq(26_266)
+      expect(Infomon.get("stat.fame")).to eq(4804958)
```

Comment:
why did you change this? `_` numbers are much more readable

Reply:
I made two mistakes.  I corrected one.  I'll change that back now to fix the second.